src/base/bittorrent/infohash.h

```diff
@@ -29,12 +29,22 @@
 #pragma once
 
 #include <libtorrent/sha1_hash.hpp>
+#include <libtorrent/version.hpp>
 
 #include <QMetaType>
 #include <QString>
 
 namespace BitTorrent
 {
+    enum class InfoHashFormat
+    {
+        V1,
+#if (LIBTORRENT_VERSION_NUM >= 20000)
+        V2,
+        Both
```

Comment:
Maybe `All`?

Reply:
I believe `Any` would be better

Reply:
>I believe `Any` would be better

It doesn't seem to fit in all cases. For example, if you request a torrent about the supported formats? Although my English is not so good that I can judge it accurately. Either way, `Both` don't look so good to me.

Reply:
The `Both` entry is removed.